README.md

```diff
@@ -43,4 +43,9 @@ yarn test
       optOut:
         - "repoName"
         - "repoName2"
+    eventWhitelist: # these are the tail of the webhook event i.e pull_request.review_requested
```

Comment:
Should just use the full identifier. Shortening just will lead to confusion.

Reply:
So the config would take the full `pull_request.review_requested` and I'll handle string splitting, is that what you mean?

Reply:
It doesn't look like this was ever updated in the readme.

Reply:
Looks like this was never fixed. Should use full webhook names.

https://github.com/ubiquity-os-marketplace/daemon-disqualifier/blob/9ca3f7ba20a48385151a55e7cf52e57cafa9cf65/tests/main.test.ts#L67